### PR TITLE
feat: fixing mypy complaint about mypy

### PR DESCRIPTION
**Pull request by SamuelLarkin**
### PR Goal? 

Fix missing default VERSION reported by `mypy`.

### Fixes? 

https://github.com/EveryVoiceTTS/EveryVoice/issues/581

### Feedback sought? 

merge approval

### Priority? 

high

### Tests added? 

no

### How to test? 

`pre-commit run --all`

### Confidence? 

good

### Version change? 

no

### Related PRs? 

None

### CODE REVIEW

**Comment by marctessier** [comment]
I am still getting the error   

```
pre-commit run --all
...
mypy.....................................................................Failed
- hook id: mypy
- exit code: 1
everyvoice/model/feature_prediction/FastSpeech2_lightning/fs2/prediction_writing_callback.py:250: error: No overload variant of "decode_tokens" of "TextProcessor" matches argument types "list[int]", "None"  [call-overload]
...
Found 1 error in 1 file (checked 4 source files)
everyvoice/model/feature_prediction/FastSpeech2_lightning/fs2/prediction_writing_callback.py:250: error: No overload variant of "decode_tokens" of "TextProcessor" matches argument types "list[int]", "None"  [call-overload]
..
Found 1 error in 1 file (checked 4 source files)
```

File: everyvoice/_version.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
-# [Valid version numbers](https://packaging.python.org/en/latest/discussions/versioning/#versioning)
-# [PEP 440 – Version Identification and Dependency Specification](https://peps.python.org/pep-0440/)
-# [Specifying Your Project’s Version](https://setuptools.pypa.io/en/latest/userguide/distribution.html)
+# [Valid version numbers](https://packaging.python.org/en/latest/discussions/versioning/#versioning)  # noqa: E501
+# [PEP 440 – Version Identification and Dependency Specification](https://peps.python.org/pep-0440/)  # noqa: E501
+# [Specifying Your Project’s Version](https://setuptools.pypa.io/en/latest/userguide/distribution.html)  # noqa: E501
 # [N!]N(.N)*[{a|b|rc}N][.postN][.devN]
-VERSION = "0.2.0a0"
+VERSION = "0.3.0a0"
```
Comment (joanise) [inline everyvoice/_version.py:5]: We didn't actually publish 0.2.0a0 yet, should stay 0.2.0a0 as the version we're currently developing until we actually publish it.

File: everyvoice/.schema/everyvoice-aligner-0.3.json
Comment (joanise) [inline everyvoice/.schema/everyvoice-aligner-0.3.json:1]: All the .schema files should be updates the 0.2 files, which have not be published to the schema registry yet.

**Review by joanise — CHANGES_REQUESTED** [review]
Seems to fix the mypy warnings, but this should not bump the version from 0.2, since we have not yet published 0.2.0a0. When you update schemas for a version that's not published yet, you need to delete the current ones manually before you rerun `everyvoice update-schemas`. Not the best Developer Experience (DX?) I know, but we don't automatically detect published vs not published yet so it just refuses to overwrite them.

File: everyvoice/_version.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
-# [Valid version numbers](https://packaging.python.org/en/latest/discussions/versioning/#versioning)
-# [PEP 440 – Version Identification and Dependency Specification](https://peps.python.org/pep-0440/)
-# [Specifying Your Project’s Version](https://setuptools.pypa.io/en/latest/userguide/distribution.html)
+# [Valid version numbers](https://packaging.python.org/en/latest/discussions/versioning/#versioning)  # noqa: E501
+# [PEP 440 – Version Identification and Dependency Specification](https://peps.python.org/pep-0440/)  # noqa: E501
+# [Specifying Your Project’s Version](https://setuptools.pypa.io/en/latest/userguide/distribution.html)  # noqa: E501
```
Comment (joanise) [inline everyvoice/_version.py:3]: With the global E501 exclusion in `.flake8` these noqa comments are not necessary.

**Review by joanise — APPROVED** [review]
Looks good now.

I would just remove the unnecessary noqa comments since we declare globally in `.flake8` that we don't care about E501.

**Pull request merged**